### ISSUE TITLE
Add Perl to available languages

### ISSUE THREAD

**Issue by kokosek** [Code-Fights/codefights-wiki#30]

**Comment by zmughal** [comment]
This seems done! But no Perl 6 yet! :-)

**Comment by kokosek** [comment]
It's not done because you can't fight using Perl yet. You can only solve challenges with this language.

**Comment by lonerz** [comment]
Well.. That's like changing over a thousand bugfix and recovery tasks (which is just a little too much you know) 👍 I'd rather have the admins be working on new features instead of changing tasks to different languages.

**Comment by kokosek** [comment]
I totally agree. But still, a request is a request. I don't mind it having infinitely low priority. ;-)
